build-tools/mono-runtimes/mono-runtimes.targets

```diff
@@ -323,9 +317,27 @@
     />
     <Touch Files="@(_InstallMonoPosixHelperOutput);@(_InstallUnstrippedMonoPosixHelperOutput)" />
   </Target>
+  <Target Name="_GetMonodocItems" DependsOnTargets="_BuildRuntimes">
+    <ItemGroup>
+      <_MonoDocCopyItems Include="@(MonoDocCopyItem->'$(_MonoOutputDir)\%(Identity)')" />
+      <_MonoDocCopyItems
+          Condition=" Exists ('$(_MonoOutputDir)\%(Identity)') "
+          Include="@(MonoDocCopyItemOptional->'$(_MonoOutputDir)\%(Identity)')"
+      />
+    </ItemGroup>
+    <ItemGroup>
```

Comment:
You can mix different items within an ItemGroup so 

```xml
<ItemGroup>
    <Foo Include="Somefile.dll" />
    <Bar Include="Someotherbarfile.dll" />
</ItemGroup>
```

MSbuild and xbuild will evaluate each "item" separately. But its sometimes useful to put items together in logical groups just to make it easier to maintain. This is probably one of those cases :)